integration/oasis/reachable_by_multiple_charge.go

```diff
@@ -1,24 +1,52 @@
 package oasis
 
 import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/Telenav/osrm-backend/integration/oasis/chargingstrategy"
 	"github.com/Telenav/osrm-backend/integration/oasis/haversine"
 	"github.com/Telenav/osrm-backend/integration/oasis/osrmconnector"
 	"github.com/Telenav/osrm-backend/integration/oasis/searchconnector"
 	"github.com/Telenav/osrm-backend/integration/oasis/stationfinder"
+	"github.com/Telenav/osrm-backend/integration/oasis/stationgraph"
 	"github.com/Telenav/osrm-backend/integration/pkg/api/oasis"
 	"github.com/Telenav/osrm-backend/integration/pkg/api/osrm/route"
 	"github.com/golang/glog"
 
 	"github.com/twpayne/go-polyline"
 )
 
-func pickChargeStationsWithEarlistArrival(oasisReq *oasis.Request, routeResp *route.Response, oc *osrmconnector.OSRMConnector, sc *searchconnector.TNSearchConnector) {
-	// chargeLocations := chargeLocationSelection(oasisReq, routeResp)
-	// for _, locations := range chargeLocations {
-	// 	c := stationfinder.CalculateWeightBetweenNeighbors(locations, oc, sc)
-	// 	sol := stationgraph.NewStationGraph(c, oasisReq.CurrRange, oasisReq.MaxRange,
-	// 		chargingstrategy.NewFakeChargingStrategy(oasisReq.MaxRange)).GenerateChargeSolutions()
-	// }
+// @todo: handle negative situation
+func generateSolutions4MultipleCharge(w http.ResponseWriter, oasisReq *oasis.Request, routeResp *route.Response, oc *osrmconnector.OSRMConnector, sc *searchconnector.TNSearchConnector) {
+	solutions := generateSolutionsWithEarlistArrival(oasisReq, routeResp, oc, sc)
+
+	w.WriteHeader(http.StatusOK)
+	r := new(oasis.Response)
+	r.Code = "200"
```

Comment:
Nit: have seen several this hardcode, use `const` instead.

Reply:
👍 Yes, better to use const value.